notebooks/00_quick_start/sar_movielens_with_azureml.ipynb

```diff
@@ -0,0 +1,668 @@
+    "\n",
+    "Then, run the experiment by submitting the estimator object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create experiment\n",
+    "EXPERIMENT_NAME = 'movielens-sar'\n",
+    "exp = Experiment(workspace=ws, name=EXPERIMENT_NAME)\n",
+    "\n",
+    "run = exp.submit(config=est)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "### 3.  Monitor remote run\n",
+    "\n",
+    "#### Jupyter widget\n",
+    "\n",
+    "Jupyter widget can watch the progress of the run.  Like the run submission, the widget is asynchronous and provides live updates every 10-15 seconds until the job completes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "ea477b629cdd4a3fad14622b28371f7c",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "_UserRunWidget(widget_settings={'childWidgetDisplay': 'popup', 'send_telemetry': False, 'log_level': 'INFO', '…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "RunDetails(run).show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Viewing run results\n",
+    "Azure Machine Learning stores all the details about the run in the Azure cloud. Let's access those details by retrieving a link to the run using the default run output. Clicking on the resulting link will take you to an interactive page."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Above cell should output similar table as below.\n",
+    "![Experiment submit output](https://recodatasets.blob.core.windows.net/images/aml_sar_output.jpg)\n",
+    "After clicking \"Link to Azure Portal\", experiment run details tab looks like this with logged metrics.\n",
+    "![Azure Portal Experiment](https://recodatasets.blob.core.windows.net/images/aml_sar_workspace.jpg)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deprovision compute resource\n",
+    "To avoid unnecessary charges, if you created compute target that doesn't scale down to 0, make sure the compute target is deprovisioned after use."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# delete () is used to deprovision and delete the AmlCompute target. \n",
+    "\n",
+    "#cpucluster.delete()"
```

Comment:
This should be `compute_target.delete()` since that is what you name the compute earlier.

Reply:
You're right